### PR TITLE
:tada: Read Only Immutable zkEVM

### PR DESCRIPTION
**Pull request by vikiival**
- **:wrench: allow node 21**
- **libs/static/src/chains.ts**
- **libs/static/src/endpoints.ts**
- **libs/static/src/indexers.ts**
- **libs/static/src/names.ts**
- **libs/static/src/services.ts**
- **libs/static/src/types.ts**
- **:wrench: wondering why does it not work**

**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Integration

## Context

- [x] Closes https://github.com/kodadot/ops-internal/issues/1036

As I have promised @yangwao to have read only version of Immutable zkEVM until Denver, well here we go ^-^. 

I have deployed new indexer called flick code is in https://github.com/vikiival/synck

I will move the repo under @kodadot after ETHDenver.

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=Fksmad33PFxhrQXNYPPJozgWrv82zuFLvXK7Rh8m1xQhe98)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prury** [comment]
nice
image

**Comment by prury** [comment]
some cors errors but loading:

image

**Comment by vikiival** [comment]
Thank you very much prury! will check that 🥹

**Comment by vikiival** [comment]
I am just getting 403 on collection detail for superpets but can be fixed later, 
How did you get the error @prury ?

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
   "engines": {
     "npm": "please-use-pnpm",
     "pnpm": "8",
-    "node": "^20"
+    "node": ">= 20"
```
Comment (Jarsen136) [inline package.json:53]: Why enable the version that is greater than 20. Should we keep the same version for every developer?

**Review by Jarsen136 — APPROVED** [review]
small question

**Comment by prury** [comment]
> I am just getting 403 on collection detail for superpets but can be fixed later, How did you get the error @prury ?

checked again and no cors now, let's go

File: package.json (hunk 1 above)
Comment (vikiival) [reply package.json:53]: I am using node 21, but agree with you 🖤

**Review by preschian — APPROVED** [review]

**Pull request merged**